src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -295,14 +295,14 @@ public void AddSource(string source)
                 // Get matching files from file pattern parser
                 matchingFiles = FilePatternParser.GetMatchingFiles(source);
             }
-            catch(TestSourceException ex)
+            catch(TestSourceException)
```

Comment:
Perhaps we can use the `catch() when` expression instead to simplify this. (doc [link](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/when#when-in-a-catch-statement))